frontend/src/routes/Governance/components/PolicyActionDropdown.tsx

```diff
@@ -150,10 +156,12 @@ export function PolicyActionDropdown(props: {
       {
         id: 'inform-policy',
         text: t('policy.table.actions.inform'),
-        tooltip: policyRemediationAction === 'inform' ? t('Already informing') : t('Inform policy'),
+        tooltip: policyRemediationAction?.includes(REMEDIATION_ACTION.INFORM)
```

Comment:
This part I am not 100% sure this so I need to double-check with reviewers

Reply:
I think the existing code was fine for this. Could you please change it back?